aws/ecs_cluster.go

```diff
@@ -0,0 +1,105 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+// tagging an ECS cluster with a given tag key-value pair and a cluster ARN
```

Comment:
NIT: might be a bit clearer to say:

```suggestion
// Tag an ECS cluster identified by the given cluster ARN with a tag that has the given key and value
```